dojos.js

```diff
@@ -1407,12 +1408,13 @@ module.exports = function (options) {
       content: content,
       code: emailCode,
       locality: emailLocality,
-      subject: emailSubject
+      subject: emailSubject,
+      subjectVariables: subjectVariables
     }, done);
   }
 
   function cmd_generate_user_invite_token (args, done) {
-    var zenHostname = args.zenHostname;
+    var zenHostname = process.env.HOSTNAME || 'localhost';
```

Comment:
Or since it's the exact same everywhere, I could just declare it at the top of the file and remove all these repeating declarations.